public/types/src/scripts/lib/choice-input.d.ts

```diff
@@ -5,5 +5,5 @@ import { ChoiceFull } from '../interfaces/choice-full';
 type MappedInputTypeToChoiceType<T extends string | InputChoice | InputGroup> = T extends InputGroup ? GroupFull : ChoiceFull;
 export declare const coerceBool: (arg: unknown, defaultValue?: boolean) => boolean;
 export declare const stringToHtmlClass: (input: string | string[] | undefined) => string[] | undefined;
-export declare const mapInputToChoice: <T extends string | InputChoice | InputGroup>(value: T, allowGroup: boolean) => MappedInputTypeToChoiceType<T>;
+export declare const mapInputToChoice: <T extends string | InputChoice | InputGroup>(value: T, allowGroup: boolean, allowRawString?: boolean) => MappedInputTypeToChoiceType<T>;
```

Comment:
This function's signature has changed and the compiled chocies.js has this function change, but the changes aren't committed to the .ts files.